### PR TITLE
Add QOL changes to Parse Server Example

### LINKED ISSUE #383: Add GitHub Actions to Parse Server Example
I think it would be helpful to:

- Define masterKey / appId via `dotenv`
- Add `.env` file but have in .gitignore
- Have a GitHub Actions workflow that runs `npm run lint` on cloud code, just like the master repo
- Have a spec folder that starts a Parse Server off index.js with a temporary database
- Have an example spec to show developers how to ensure their cloud code is running as expected

### PR DESCRIPTION
**Pull request by dblythy**
Adds:

- Example GitHub workflow
- Testing utility that imports `index.js` and boots up a temporary DB, and destroys after tests are complete 
- Example unit test
- Example code linting

Closes #383

Feel free to let me know your thoughts / what can be improved.

@mtrezza @davimacedo @Moumouls

### CODE REVIEW

**Review by davimacedo — APPROVED** [review]
LGTM!

**Comment by davimacedo** [comment]
@dblythy it looks the action is failing after merged. Would you take a look?

**Comment by dblythy** [comment]
Yes sorry, I added a failing test as an example of what happens when a test fails. Is this a bad idea?

**Comment by davimacedo** [comment]
Yes, I think so. I haven't noticed that as well. But only passing tests would be better.

File: package.json [hunk 1]
```diff
@@ -9,15 +9,29 @@
...
+  "devDependencies": {
+    "eslint": "^7.17.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "babel-eslint": "^10.1.0",
+    "jasmine": "^3.6.3",
+    "mongodb-runner": "^4.8.0",
+    "bluebird": "^3.7.2"
```
Comment (mtrezza) [inline package.json:35]: @dblythy I am wondering now why this repo suddenly depends on `bluebird`? Is this really necessary?

Comment (dblythy) [reply package.json:35]: Yes, it is used in the [test runner](https://github.com/parse-community/parse-server-example/blob/master/spec/utils/test-runner.js#L1).

Note that this from a different repo: [Parse Test Runner](https://github.com/AmpMe/parse-server-test-runner/blob/master/src/index.js)

Comment (mtrezza) [reply package.json:35]: We are also not using bluebird in parse-server, so I wonder why we need it here?

Comment (dblythy) [reply package.json:35]: I'm not sure to be honest. I just used the test runner that had already been created. I can work on refactoring.

Comment (mtrezza) [reply package.json:35]: Well, maybe the way parse-server runs its tests is more up-to-date. The code referenced is from 2017.

Comment (dblythy) [reply package.json:35]: Yes, I agree. I'll work on it.

Comment (mtrezza) [reply package.json:35]: Cool! I added #390.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [1.2.0](https://github.com/parse-community/parse-server-example/releases/tag/1.2.0)

**Pull request merged**